ClippingBezier/UIBezierPath+Trimming.m

```diff
@@ -174,6 +174,75 @@ - (UIBezierPath *)bezierPathByTrimmingToLength:(CGFloat)trimLength
     return newPath;
 }
 
+- (UIBezierPath *)bezierPathWithRange:(NSRange)range
```

Comment:
if `elementIndexWhereOverlapOccursWithClosedPath` can go then i think this one can too